pkg/metrics/smart/metrics.go

```diff
@@ -31,6 +31,33 @@ type MetricsData struct {
 	// blockDeviceTemperature is the temperature of the the blockdevice if it is reported
 	blockDeviceCurrentTemperature *prometheus.GaugeVec
 
+	// blockDevicehighestTemperature is the highest temperature of the the blockdevice if it is reported
+	blockDeviceHighestTemperature *prometheus.GaugeVec
+
+	// blockDevicelowestTemperature is the lowest temperature of the the blockdevice if it is reported
+	blockDeviceLowestTemperature *prometheus.GaugeVec
+
+	// blockDeviceRotationalLatency is rotational latency of a block device
+	blockDeviceRotationalLatency *prometheus.GaugeVec
+
+	// blockDeviceCapacity is capacity of block device
+	blockDeviceCapacity *prometheus.GaugeVec
+
+	// blockDeviceRotationRate is rotation rate of a block device in RPM
+	blockDeviceRotationRate *prometheus.GaugeVec
```

Comment:
The rotation rate is a fixed value. Should we collect it from here? Or use the value available in etcd?

Reply:
I think it's better if it's here as cloud providers' virtual disks show a change in performance when there's a change in capacity.

Reply:
The rotation rate is something like 5400 RPM, 7200 RPM right? Even if capacity changes that value won;t change.

Reply:
Please check out the section "Performance by disk size". https://cloud.google.com/compute/docs/disks/performance .

Reply:
The above is given for different sized SSDs to match performance  of a 7200 RPM drive. The performance comparison given in there and the actual RPM are different contexts. 

Drive RPM is something that you *cannot* change on the fly, even with cloud volumes. Its a hardware set value. And AFAIK seachest will not report this value for cloud volumes because they are virtual drives. It can be obtained only for physical disks.

Reply:
That would also mean rotational latency is constant. Is it okay if we move these two fields when we raise a PR for static exporter? Maybe comment it out for now?

Reply:
> That would also mean rotational latency is constant.

Yes.

> Is it okay if we move these two fields when we raise a PR for static exporter?

you can remove those fields, no need to keep them commented. and move it to the static exporter.